src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: all $transitionTime;
```

Comment:
don't use it like that, because it's the same as transition-duration:  $transitionTime
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/223094425-1bfef1f2-98e5-4cd5-8431-0a80e6f3c278.png)

```suggestion
    transition: transform $transitionTime, ...;
```